processing/db_handler.py

```diff
@@ -164,6 +197,62 @@ def get_filename_checksum_lookup(self, conn):
                 checksums[fname] = r.get('file_checksum')
         return checksums
 
+    def get_upload_file_info(self, conn, upload_id):
+        upload_file_info = {}
+        query = SQL('SELECT * FROM input_interface.data_upload_log '
+                    'WHERE log_id = %s')
+        with conn.cursor(cursor_factory=RealDictCursor) as cursor:
+            cursor.execute(query, (upload_id,))
+            for r in cursor:
+                upload_file_info['upload_id'] = r.get('log_id')
+                upload_file_info['site_id'] = r.get('site_id')
+                upload_file_info['upload_comment'] = r.get('upload_comment')
+                upload_file_info['data_file'] = r.get('data_file')
+        return upload_file_info
+
+    def register_format_qaqc(self, upload_id: int, process_timestamp: str,
+                             site_id: str,
+                             prior_process_id: Optional[int] = None,
+                             zip_process_id: Optional[int] = None) -> int:
+
+        process_code_version, db_config = self._read_config()
+        processor_user_id = socket.gethostname()
+
+        field_names = ('process_type_id, process_timestamp, upload_id, '
+                       'site_id, processor_user_id, processing_code_version')
+        values = [1, process_timestamp, upload_id, site_id, processor_user_id,
```

Comment:
The hard coded 1 needs some comment here. I would prefer if the value is done via the actual source.

Reply:
done.

Along with data source type in file_fixer.

Reply:
never mind on file_fixer -- got carried away.